spec/ttml1.xml

```diff
@@ -13244,6 +13243,53 @@ shown in <specref ref="fragment-streaming-graphic"/>.</p>
 <p>This specification does not define a transport buffer model or a decoder
 capabilities model.</p>
 </note>
+</div2>
+<div2 id="temporal-fragmentation">
+<head>Temporal fragmentation</head>
+<p>Another possible means by which TTML Content may be streamed is to
+partition a <emph>Document Instance</emph>'s information set into
+temporally bound fragments, each of which is itself a document instance that contains all the
+front matter (head) and content required to present it, where the temporal interval for each
+fragment is constrained, either by the <emph>Document Interchange Context</emph> or by the 
+<emph>Document Processing Context</emph> or by both, such that temporal overlaps with other 
```

Comment:
Change ``<emph>Document Processing Context</emph>`` to a link to terminology section (with lower case).